go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/go-logr/logr v0.3.0
 	github.com/googleapis/gnostic v0.5.3 // indirect
 	github.com/k8snetworkplumbingwg/network-attachment-definition-client v1.1.0
+	github.com/mattn/goveralls v0.0.8 // indirect
```

Comment:
why you need goveralls? We don't integrated  Coveralls.io  in network operator at the moment

Reply:
Need to investigate why Travis CI add it. Let's remove it for now